src/query/parser/promql/resolve_scalar_test.go

```diff
@@ -85,3 +86,32 @@ func TestScalarResolver(t *testing.T) {
 		})
 	}
 }
+
+var stringResolverTests = []struct {
+	funcString string
+	expected   string
+}{
+	{"(((\"value\")))", "value"},
```

Comment:
:+1:

Reply:
Sure, I guess ` are mostly used for multiline strings but I think it makes sense here as well.